pkgs/applications/video/lbry/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv
+, dpkg
+, fetchurl
+, at-spi2-core
+, atomEnv
+, ffmpeg
+, gtk3
+, libdrm
+, libpulseaudio
+, libuuid
+, makeWrapper
+, mesa
+, xdg_utils
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lbry-desktop";
+  version = "0.48.2";
+
+  src = fetchurl {
+    url = "https://github.com/lbryio/lbry-desktop/releases/download/v${version}/LBRY_${version}.deb";
+    sha256 = "0kqyiv9lfck9zzhiqpbvmlbyk09hajcaa2bpk1i3w03yk548an6z";
+  };
+
+  rpath = stdenv.lib.makeLibraryPath [
+    at-spi2-core
+    libdrm
+    libpulseaudio
+    libuuid
+    mesa
+    xorg.libXScrnSaver
+    xorg.libxcb
+  ] + ":${atomEnv.libPath}";
+
+  buildInputs = [
+    ffmpeg
+    gtk3
+  ];
+
+  nativeBuildInputs = [ dpkg makeWrapper ];
```

Comment:
The lists of dependencies, including the attribute “buildInputs” should preferably come before lists of dependencies’ “nativeBuildInputs” attribute in the expression.

Reply:
I don't think I know what you're asking for. Do you mean put `rpath` below `nativeBuildInputs`, or maybe also put the packages used in `rpath` in `buildInputs`?

Reply:
I don't feel this is a reasonable complaint.